pinot-common/src/main/java/org/apache/pinot/sql/parsers/CalciteSqlParser.java

```diff
@@ -752,6 +754,8 @@ private static Expression compileFunctionExpression(SqlBasicCall functionNode) {
       case OR:
         return compileOrExpression(functionNode);
       // BETWEEN and LIKE might be negated (NOT BETWEEN, NOT LIKE)
+      case EXTRACT:
```

Comment:
Do we need to handle this separately? Seems it is the same as the default handling logic

Reply:
Thanks for the insight. The default case was handling the logic.